gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -266,16 +266,45 @@ private Gradebook getGradebook(String siteId) {
 	}
 	
 	/**
-	 * Get a list of assignments in the gradebook in the specified site that the current user is allowed to access, sorted by sort order
-	 * 
+	 * Get a list of assignments in the gradebook in the specified site that the current user is allowed to access, sorted by default sort order
+	 *
 	 * @param siteId the siteId
 	 * @return a list of assignments or null if no gradebook
 	 */
 	public List<Assignment> getGradebookAssignments(String siteId) {
```

Comment:
May need an override of this method to allow both siteId and the boolean, just so both forms are available. Alternatively, we can kill the ones we aren't using. The methods in the business service that accept the siteId are needed for any entity providers as they don't have a site context, but if they aren't being used at the moment they could just be removed for simplicity.